packages/astro/src/@types/astro.ts

```diff
@@ -2186,6 +2186,30 @@ export interface AstroUserConfig {
 		 */
 		serverIslands?: boolean;
 
+		/**
+		 * @docs
+		 * @name experimental.contentCollectionIntellisense
+		 * @type {boolean}
+		 * @default `false`
+		 * @version 4.14.0
+		 * @description
+		 *
+		 * Enables the generation of files required for content intellisense for Astro content files.
+		 *
+		 * When enabled, this feature will add JSON schemas to the `.astro` directory in your project, which can be used by the Astro language server to provide intellisense inside content files (`.md`, `.mdx`, `.mdoc`).
```

Comment:
```suggestion
		 * Enables Intellisense features (e.g. code completion, quick hints) for your content collection entries in compatible editors.
		 *
		 * When enabled, this feature will generate and add JSON schemas to the `.astro` directory in your project. These files can be used by the Astro language server to provide Intellisense inside content files (`.md`, `.mdx`, `.mdoc`).
```
Intellisense seems to be capitalized as a proper noun. 

Just a thought: also thinking we could choose to highlight what the feature unlocks, vs the fact that files are generated. (I made up the "compatible editors" because now we're claiming not just the file generation, but actually promising Intellisense. If this isn't the right way to frame it, could be something like "in editors that support the Astro language server.")